x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/utils/connector_status_helpers.ts

```diff
@@ -52,17 +60,23 @@ export function connectorStatusToText(
 }
 
 export function connectorStatusToColor(
-  connectorStatus: ConnectorStatus,
+  connector: Connector,
   hasIndexName: boolean
```

Comment:
Since we're passing the whole connector now instead of just the status should we remove this parameter and just do the `!!connector?.index_name` in the function?

Reply:
Yes! I saw that and forgot to change. Will update and merge after